src/Abp/Configuration/SettingManager.cs

```diff
@@ -27,6 +28,11 @@ public class SettingManager : ISettingManager, ISingletonDependency
         /// </summary>
         public ISettingStore SettingStore { get; set; }
 
+        /// <summary>
+        /// Reference to multi tenancy configuration.
+        /// </summary>
+        public IMultiTenancyConfig MultiTenancyConfig { get; set; }
```

Comment:
Use constructor injection since there is no default/null value for `IMultiTenancyConfig`  (like DefaultConfigSettingStore.Instance)